openfisca_us/variables/finance/benefit/snap.py

```diff
@@ -156,9 +156,15 @@ def formula(spm_unit, period, parameters):
         )
 
 
-class is_disabled_or_elderly_for_snap(Variable):
+class is_usda_disabled(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Indicates that a person is defined as disabled based on the USDA definition"
```

Comment:
Could we add a label? And below

Reply:
and below?

Reply:
Just in the other variables too (`is_usda_elderly` etc.). Thanks!